env/git.go

```diff
@@ -112,26 +112,21 @@ func GitSHA(path string) (string, error) {
 }
 
 var GitBlob = func(path string, commit *object.Commit) (string, error) {
-	if commit == nil {
-		blob, err := fallbackBlob(path)
+	if commit != nil {
+		if file, err := commit.File(path); err == nil {
```

Comment:
i think i'm going to rewrite this as: 
```
file, err := commit.File(path)
if err == nil {
  // return git blob
} 
```
basically if we didn't get any error when trying to find the file from git, we can get the blob. Otherwise just call the fallback method.

sorry that i didn't explain better the issue.